swarm.cabal

```diff
@@ -130,7 +131,32 @@ library
                       yaml                          >= 0.11 && < 0.12,
     hs-source-dirs:   src
     default-language: Haskell2010
-    default-extensions: StrictData
+    default-extensions:
```

Comment:
perhaps we should move default-language and default-extensions to a new common used by every target. Like that tests and executables uses the same extensions as the lib.